puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
The only parameters I am adding at this time are:
  1. netapp_hostname
  2. netapp_login
  3. netapp_password
  4. netapp_server_port
  5. netapp_storage_family
  6. netapp_transport_type
  7. netapp_storage_protocol
  8. netapp_nfs_shares
  9. netapp_nfs_shares_config
  10. netapp_volume_list
  11. netapp_vfiler
  12. netapp_vserver
  13. netapp_controller_ips
  14. netapp_sa_password
  15. netapp_storage_pools

Those should be the only param names that you see in this PR.  Is there another location that these need to be put?